scripts/microbe_stage/microbe.lua

```diff
@@ -313,6 +324,10 @@ function Microbe:__init(entity)
     end
     if not self.microbe.initialized then
         self:_initialize()
+        if in_editor == nil then
```

Comment:
I remember having a very annoying bug, probably due to missing something obvious, which I fixed by making this check if the parameter exists. I can revisit it though.